src/Query.php

```diff
@@ -103,7 +103,7 @@ private function datetime_to_utc_string($datetime) {
         if($datetime instanceof \DateTime) {
             $datetime_copy = clone $datetime;
             $datetime_copy->setTimezone(new \DateTimeZone('UTC'));
-            return $datetime_copy->format('Y-m-d\TH:i:s');
+            return $datetime_copy->format('Y-m-d\TH:i:s\Z');
```

Comment:
We could name our own helper method (`datetime_to_utc_string`) to something with iso8601, to make our intentions more clear.